app/controllers/publishers/organisations/email_controller.rb

```diff
@@ -0,0 +1,26 @@
+class Publishers::Organisations::EmailController < Publishers::OrganisationsController
+  def edit
+    organisation
+    @email_form = Publishers::Organisation::EmailForm.new(email: organisation.email)
+  end
+
+  def update
+    @email_form = Publishers::Organisation::EmailForm.new(email_form_params)
+
+    if @email_form.valid?
+      organisation.update(email: params[:publishers_organisation_email_form][:email])
```

Comment:
Minor thing, I think perhaps we should use `organisation.update(email: @email_form.email)` here